include/word.h

```diff
@@ -0,0 +1,27 @@
+#ifndef INCLUDE_WORD_H_
+#define INCLUDE_WORD_H_
+#include <stdio.h>
+#include <strings.h>
+#include "dictionary.h"
+
+/*
+ * valid_word: checks that word is in dictionary
+ * parameters: word, dictionary
+ * returns: int (EXIT_SUCCESS OR FAILURE)
+ * EXIT_FAILURE if it is not in array
+ * EXIT_SUCCESS if it is in dictionary
+ */
+int valid_word(dict_t *dict, char* word);
+
+/*
+ * generate_suggestions: receives a dictionary, word, and an allocated array of suggestions
```

Comment:
Again, out-parameters suck. This should rather return a linked list.

Reply:
will change to return list

Reply:
this is a hard coded function for now, dawson has the actual version of the function

Reply:
I think the issue is that the suggestion generation exists in suggestion.c, which is another function entirely and one that we haven't authored. An integrated version does exist in firat/jae/word, which I was planning on doing a PR as soon as this one was over. Should cherry-pick still happen?

Reply:
Hey, I would be careful about adding in the suggestions function stuff- it happens to be about 1000 lines or so and all of that has been moved to the tries team and suggestion.c will become a wrapper for the API call. This should be returning an allocated char** containing a simple list of words. Also, it should only take in a dictionary and a word to generate suggestions on.